api/Window.json

```diff
@@ -5330,6 +5330,58 @@
           }
         }
       },
+      "orientationchange_event": {
+        "__compat": {
+          "description": "<code>orientationchange</code> event",
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/Window/orientationchange_event",
+          "support": {
+            "chrome": {
+              "version_added": false
+            },
+            "chrome_android": {
+              "version_added": true
+            },
+            "edge": {
+              "version_added": false
+            },
+            "edge_mobile": {
+              "version_added": true
+            },
+            "firefox": {
+              "version_added": false
+            },
+            "firefox_android": {
+              "version_added": "44"
+            },
+            "ie": {
+              "version_added": false
+            },
+            "opera": {
+              "version_added": false
+            },
+            "opera_android": {
+              "version_added": true
+            },
+            "safari": {
+              "version_added": false
+            },
+            "safari_ios": {
+              "version_added": true
+            },
+            "samsunginternet_android": {
+              "version_added": true
+            },
+            "webview_android": {
+              "version_added": true
+            }
+          },
+          "status": {
+            "experimental": false,
+            "standard_track": true,
+            "deprecated": true
```

Comment:
Is this really deprecated?

Reply:
I believe so. It is specified in the compat spec and I think the real thing to use is https://developer.mozilla.org/en-US/docs/Web/Events/deviceorientation.